tests/cross_functional/ui/test_create_pool_block_pool.py

```diff
@@ -118,6 +122,19 @@ def test_create_delete_pool(
         # Getting IO results
         get_fio_rw_iops(self.pod_obj)
 
+        # Checking the raw capcity is loaded on the UI or not.
+        blockpool_ui_object = BlockPoolUI()
+        if not blockpool_ui_object.pool_raw_capacity_loaded(self.pool_name):
+            raise BlockPoolRawCapacityNotLoaded(
+                f"The Raw Capacity for blockpool {self.pool_name} is not loaded in UI."
+            )
+
+        # Cross checking the raw capacity of the blockpool between CLI and UI
+        if not blockpool_ui_object.cross_check_raw_capacity(self.pool_name):
+            raise BlockPoolRawCapacityNotCorrect(
```

Comment:
Shivam, I am not quite agree that these should bring errors/exceptions. 
I thing it would be better to throw assertion here, to increase number of Fails but not Errors in report.

@petr-balogh since you're triggering regression, what do you think?
Will be good to standardise this for all.

Reply:
It will not have effect on errors/failures in pytest. But I agree that in the test itself it's better to just use simple assert a >=b , "Assert error message if not true" instead of creating specific exception for the test.

Reply:
updated